integration/docker/Dockerfile.fuse

```diff
@@ -50,6 +50,8 @@ RUN if [ ${ALLUXIO_USERNAME} != "root" ] \
       mkdir -p /journal && \
       chown -R ${ALLUXIO_UID}:${ALLUXIO_GID} /opt/* /journal && \
       chmod -R g=u /opt/* /journal; \
+      mkdir /alluxio-fuse && \
```

Comment:
I am still curious why previously docker and kubernetes can launch Alluxio fuse containers without creating the alluxio-fuse folder?

Reply:
Previously this worked because the container ran as `root` (uid 0), so the directory could be created at runtime. Now, because we're running as `alluxio`, permissions to `/` are denied. So we create the folder and `chown` at container creation time